### ISSUE TITLE
Accessibility from Lighthouse!

### ISSUE THREAD

**Issue by teenie-quaggard** [fac-15/weatherTunes#21]
1. Aria labels = 'role' not valid
2. Not enough color contrast (Failing elements: 

<p>Tunes to match the weather in your location</p>
--
<input type="submit" name="submit" value="Submit">
)
3. id elements are not unique
4.iframe elements do not have a title

**Comment by martingaston** [comment]
Also on accessibility: when you're tabbing through the page on first load you'll hit the (hidden) box reserved for the video, rather than the input form.
